src/Orchard.Web/Modules/Orchard.Workflows/Orchard.Workflows.csproj

```diff
@@ -70,7 +70,7 @@
       <RequiredTargetFramework>3.5</RequiredTargetFramework>
     </Reference>
     <Reference Include="System.Data.DataSetExtensions" />
-    <Reference Include="System.Net.Http" />
+    <Reference Include="System.Net.Http, Version=4.2.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a" />
```

Comment:
This comment is relevant for all the changes across various files that have a change related to System.Net.Http:
It started with the static Razor compilation throwing a warning (which is now intentionally configured to be an error) that there are conflicting references with a different target framework. With some experimentation it turned out that pinning the 4.2.0.0 version fixes all the build warnings and the code that actually uses it (e.g., the WebRequestActivity in Workflows) still works. On the other hand, this and the other modules depending on Workflows also need this reference set in their web.config.

System.Net.Http is proven to be problematic in some cases - not sure if directly related, but still an interesting read: https://github.com/dotnet/runtime/issues/65005.